lib/membrane_mp4/movie_fragment_box.ex

```diff
@@ -42,6 +48,13 @@ defmodule Membrane.MP4.MovieFragmentBox do
         data_offset: 0
       })
 
+    config =
+      if Map.get(config, :support_b_frames, false) == true do
```

Comment:
Is this parameter really optional? If not:
```suggestion
      if config.support_b_frames do
```

Reply:
Hmm,  as far as I know, `MovieFragmentBox.assemble/1` can be used not only with CMAF module, so the field from `CMAF.__struct__` will not necessary be present...however, according the the `MovieFragmentBox.assemble/1` typespec we are expecting `support_b_frames` field, so I believe we can use your suggestion :D

Reply:
done